OData/src/System.Web.OData/OData/Query/ODataQueryOption.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Web.OData.Query
+{
+    /// <summary>
+    /// Defines the abstraction for OData query option.
+    /// </summary>
+    public abstract class ODataQueryOption
```

Comment:
the name "ODataQueryOption" is very confusing. Meantime, it's better to find a name unlike "ODataQueryOptions".